docs/_source/conf.py

```diff
@@ -209,7 +209,9 @@
     ('py:class', 'TextIO'),
     ('py:class', '_io.StringIO'),
     ('py:class', 'yaml.loader.SafeLoader'),
-    ('py:class', 'yaml.dumper.Dumper')
+    ('py:class', 'yaml.dumper.Dumper'),
+    ('py:class',  'cfn_tools.odict.ODict'),
+    ('py:class',  'T_Container')
```

Comment:
There were some warnings being output by Sphinx about it not knowing how to resolve docs for these classes.